pkg/assets/create/creater.go

```diff
@@ -74,19 +69,15 @@ func EnsureManifestsCreated(ctx context.Context, manifestDir string, restConfig
 		if needDiscoveryRefresh {
 			mapper, err = fetchLatestDiscoveryInfoFn(dc)
 			if err != nil {
-				if options.Verbose {
-					fmt.Fprintf(options.StdErr, "[#%d] failed to fetch discovery: %s\n", retryCount, err)
-				}
+				options.Logger.Logf("[#%d] failed to fetch discovery: %s", retryCount, err)
```

Comment:
project standard upstream and downstream is fmt.printf for CLI and glog for everything else.  The previous code is in keeping.

Reply:
> project standard upstream and downstream is fmt.printf for CLI and glog for everything else. The previous code is in keeping.

But this is library code, so I'd expect glog over fmt, right?

Reply:
@wking depends on the caller of this code. So far it is cluster-bootstrap which is a command we run during the bootstrap (so it is CLI ;-). I'm fine switching this to glog, was trying to avoid importing glog because I know that some components/cli have problem with parsing glog flags (so wanted to avoid that if not necessary).

Reply:
> depends on the caller of this code. So far it is cluster-bootstrap which is a command we run during the bootstrap (so it is CLI ;-).

I don't think code in library-go should have API choices hinge on "we know all of our consumers, and they're not libraries themselves".  Is there a reason that approach is not as brittle as it seems?

> I'm fine switching this to glog, was trying to avoid importing glog because I know that some components/cli have problem with parsing glog flags (so wanted to avoid that if not necessary).

Like openshift/installer#1181.  This is exactly why I prefer logging interfaces, but it sounds like that's a no-go?